medusa/post_processor.py

```diff
@@ -87,7 +89,9 @@ def __init__(self, file_path, nzb_name=None, process_method=None, is_priority=No
 
         self.manually_searched = False
 
-        self.item_resources = {'file name': self.file_name, 'relative path': self.rel_path, 'nzb name': self.nzb_name}
+        self.item_resources = OrderedDict({'file name': self.file_name,
```

Comment:
This way you have the same issue since you're using a regular dict to Instantiate an ordered one